service-front/app/features/context/Integration/LpaContext.php

```diff
@@ -1571,12 +1574,13 @@ public function iAmShownTheDetailsOfAnLPA()
                     json_encode(
                         [
                             'donor' => [
-                                'uId' => $this->lpa['donor']['uId'],
-                                'firstname' => $this->lpa['donor']['firstname'],
-                                'middlenames' => $this->lpa['donor']['middlenames'],
-                                'surname' => $this->lpa['donor']['surname'],
+                                'uId'           => $this->lpa['donor']['uId'],
+                                'firstname'     => $this->lpa['donor']['firstname'],
+                                'middlenames'   => $this->lpa['donor']['middlenames'],
+                                'surname'       => $this->lpa['donor']['surname'],
```

Comment:
```suggestion
                                'uId'         => $this->lpa['donor']['uId'],
                                'firstname'   => $this->lpa['donor']['firstname'],
                                'middlenames' => $this->lpa['donor']['middlenames'],
                                'surname'     => $this->lpa['donor']['surname'],
```
Always the longest one dictating the length of a line.